tests/Traits/RealCacheImplementationTrait.php

```diff
@@ -24,11 +23,7 @@ protected function tearDown(): void
     private function getCache(): CacheInterface
     {
         if ($this->_cache === null) {
-            $this->_cache = new FilesystemCachePool(
-                new Filesystem(
-                    new Local(sys_get_temp_dir() . '/unleash-sdk-tests')
-                )
-            );
+            $this->_cache = new Psr16Cache(new ArrayAdapter());
```

Comment:
Similarly, we can use `symfony/cache` here too, which is especially useful now that `cache/filesystem-adapter` may not be installed when the test suite runs.